bin/rock

```diff
@@ -211,6 +211,28 @@ done
 
 case "$1" in
 
+(diag)
+	COMMAND=${1}
```

Comment:
For each of these lines, let's add an echo to the log of exactly what we're running g so it's obvious when reading the log